docs/content/latest/yugabyte-platform/manage-deployments/high-availability.md

```diff
@@ -0,0 +1,187 @@
+
+    <br/>
+
+    ![Standby instance type](/images/yp/high-availability/standby-configuration.png)
+
+1. Enter this platform’s IP address or hostname (including the HTTP or HTTPS protocol prefix, and port if you're not using the default of 80 or 443).
+
+1. Paste the shared authentication key from the active platform into the Shared authentication key field. Double-check to make sure the keys match.
+
+    <br/>
+
+    ![Shared key field](/images/yp/high-availability/shared-key-field.png)
+
+1. **On the active platform**, click the Instance Configuration tab. Click Add Instance, enter the new standby's IP address or hostname (including the HTTP or HTTPS protocol prefix, and port if you're not using the default of 80 or 443), and click Continue.
+
+    <br/>
+
+    ![Add an instance](/images/yp/high-availability/add-standby-instance.png)
+
+1. **On the new standby platform**, wait for a replication interval (you set this when you [set up the active instance](#set-up-the-active-platform)), then refresh the page. The other instances in the HA cluster should now appear in the list of instances.
+
+    <br/>
+
+    ![Instance list](/images/yp/high-availability/instance-configuration.png)
+
+    If you click on other tabs, notice that they are greyed out. You can only use those tabs on the active instance.
+
+**Your standby instances are now set up**. The next section details how to [promote a standby platform instance to active](#promote-a-standby-platform-to-active), and the final section explains how to [delete a standby instance](#remove-a-standby-platform).
+
+### Promote a Standby Platform to Active
+
+1. On the standby platform you wish to promote to active, click the Make Active button in the upper-right corner of the "Replication Configuration" tab.
+
+    <br/>
+
+    ![Standby instance replication configuration](/images/yp/high-availability/standby-replication-configuration.png)
+
+1. Select the backup you want to restore (we strongly recommend you **choose the most recent backup** in the vast majority of cases!), and ensure the Confirm promotion box is checked.
+
+    <br/>
+
+    ![Backup selection](/images/yp/high-availability/restore-backup.png)
+    <br/>
+
+    The restore should take only a few seconds, and you'll be logged out when it's finished.
+
+1. To log back in, use the user credentials that you had configured on the previously active platform.
+
+    <br/>
+
+    Once you're logged in, you should see that all of the data has been restored into the platform including universes, users, metrics, alerts, task history, cloud providers, and so on.
+
+### Remove a Standby Platform
+
+To remove a standby platform from a high-availability cluster, you remove it from the active platform's list, and then delete the configuration from the platform to be removed.
+
+1. **On the standby platform** you wish to remove from the high-availability cluster, click the Delete Configuration button on the Admin tab, and confirm the deletion.
```

Comment:
Nit: Swap steps (1) and (2) here -> i.e. you want to delete it from the active first and then delete the whole config from the standby

Reply:
Done!